pyEpiabm/pyEpiabm/covidsim_helpers.py

```diff
@@ -0,0 +1,54 @@
+#
+# Helper functions based on Covidsim code.
+#
+from .person import Person
+
+
+class CovidsimHelpers:
+    '''
+    Class to create helper functions from Covidsim code to aid
+    comparison to Covidsim.
+    '''
+
+    @staticmethod
+    def calc_house_inf(infector: Person, timestep: int):
+        """Calculate the infectiveness of a household.
+
+        :param infector: Infector.
+        :type infector: Person
+        :param timestep: Current simulation timestep
+        :type timestep: int
+        :return: infectiveness
+        :rtype: float
+        """
+        return 1
+
+    @staticmethod
+    def calc_house_susc(infector: Person, infectee: Person, timestep: int):
+        """Calculate the susceptibility of a household.
+
+        :param infector: Infector.
+        :type infector: Person
+        :param infectee: Infectee.
+        :type infectee: Person
+        :param timestep: Current simulation timestep
+        :type timestep: int
+        :return: susceptibility
+        :rtype: float
+        """
+        return CovidsimHelpers.calc_person_susc(infector, infectee, timestep)
+
+    @staticmethod
+    def calc_person_susc(infector: Person, infectee: Person, timestep: int):
```

Comment:
I know we haven't implemented them yet, but do we anticipate that these functions will have explicit time dependence (and that an individual's susceptibility depends on who is infecting them - not clear if this is an interaction property or a property of an individual? If it is a property of an individual, do they have one for each possible infector?)

Not a big issue but maybe worth being consistent now for what we plan to add later

Reply:
Say we have an immune resistance decrease fucntion which is pulled in within the inf/susc function and needs that time dependence

Reply:
Ooh okay yep that makes sense, and we could add in some internal variable to store infection time to convert absolute simulation time to relative infection time.

Still not sure about the infector argument though, I thought susceptibility was a property of an individual not a potential infection opportunity, so why could it depend on the infector object? Think it should only take one person object as an argument (but happy to be convinced otherwise!)